api/api/projects_api_test.go

```diff
@@ -369,14 +376,64 @@ func TestListProjects(t *testing.T) {
 
 func TestUpdateProject(t *testing.T) {
 	testCases := []struct {
-		desc             string
-		projectID        models.ID
-		existingProject  *models.Project
-		expectedResponse *Response
-		body             interface{}
+		desc                  string
+		projectID             models.ID
+		existingProject       *models.Project
+		expectedResponse      *Response
+		expectedMessage       map[string]interface{}
+		body                  interface{}
+		updateProjectEndpoint string
+		updateProjectPayload  string
+		updateProjectResponse string
+		labelsBlacklist       map[string]bool
```

Comment:
Can we have the `UpdateProjectConfig` class specified in the test case struct? That way we don't have to keep redeclaring the individual nested fields in each test case, and we can avoid reinitialising this `UpdateProjectConfig` struct here: https://github.com/caraml-dev/mlp/pull/99/files#diff-abc4bd2a1d3aed82b2a1e745979b177340e91c9d48f008647a35595a40ac4809R574